format_implementing.md

```diff
@@ -0,0 +1,92 @@
+# Format Implementation Guide
+
+If you're a launcher developer, this document can help you by providing useful tips and recommended practices for implementing the MODIP Format.
+
+---
+
+## Rendering HTML
+
+The MODIP Format Specification allows the usage of HTML in certain fields, such as `description` and `changelog`. HTML is widely used and meets many of the requirements needed for a field like `description`. However, HTML can be **very dangerous**. Take caution when displaying content using HTML. It's recommended to sanitize your HTML according to the rules listed below.
+
+### Disallowed Tags
+
+You should never attempt to render any of the following tags: `script`, `style`, `link`, `title`, `button`, `form`, `input`, `meta`, `option`, `textarea`, and `select`.
+
+### iframes
+
+`iframes` can be used, but it's recommended to only render them when the hostname is from a trustworthy source. Trustworthy sources are up to implementation by the launcher.
```

Comment:
So in reality they can't be used.

Reply:
It really depends on what the community does. If most launchers decide to allow youtube.com for example, then you can assume it's safe to use a youtube.com embed. A list of "recommend trustworthy sources" could maybe be added, but I don't know if that's something MODIP should decide.